src/styles/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { ButtonBase, withStyles } from '@material-ui/core';
+import { styled } from '@mui/material';
```

Comment:
Webchat already exports `styled` and you can get it with the following code:
``` 
import * as FlexWebChat from '@twilio/flex-webchat-ui';
const { styled } = FlexWebChat;
```

Reply:
@murilovmachado Thank you very much for this.

Reply:
I'd go with just styled components based on what Murilo mentioned - https://styled-components.com 
Correct me if I'm wrong, @stephenhand @GPaoloni

Reply:
@mythilytm That's correct. Thank you.